test/lib/fsInterface.js

```diff
@@ -1,13 +1,15 @@
-const test = require("ava");
-const {promisify} = require("util");
-const Buffer = require("buffer").Buffer;
-const path = require("path");
-const fs = require("fs");
-const ui5Fs = require("../../");
-const fsInterface = ui5Fs.fsInterface;
-const MemAdapter = ui5Fs.adapters.Memory;
-const FsAdapter = ui5Fs.adapters.FileSystem;
-const Resource = ui5Fs.Resource;
+import test from "ava";
+import {promisify} from "node:util";
+import {Buffer} from "node:buffer";
+import path from "node:path";
+import {fileURLToPath} from "node:url";
+import fs from "node:fs";
+const stat = promisify(fs.stat);
+const readFile = promisify(fs.readFile);
```

Comment:
`readFile` is already available as promise returning method. Use `import {readFile} from "node:fs/promises";`

Reply:
Done